components/kyma-operator/cmd/operator/main.go

```diff
@@ -104,3 +111,16 @@ func getClientConfig(kubeconfig string) (*rest.Config, error) {
 	}
 	return rest.InClusterConfig()
 }
+
+func parseBackoffIntervals(backoffIntervals string) ([]uint, error) {
+	backoffIntervalsParsed := []uint{}
+	for _, interval := range strings.Split(backoffIntervals, ",") {
+		parsedInterval, err := strconv.ParseUint(strings.TrimSpace(interval), 10, 32)
```

Comment:
Suggestion: make it a bit more user-friendly and handle trailing commas.
At the moment, it causes an error.
```
go run cmd/operator/main.go --backoffIntervals="1,2,3,"
2020/02/26 10:10:57 Starting operator...
2020/02/26 10:10:57 Unable to parse backoff intervals configuration. Error: strconv.ParseUint: parsing "": invalid syntax
```

Reply:
done, now an error for trailing commas will not be returned and in case of not proper uint value the message should be more readable for the user